src/json/text.json

```diff
@@ -1129,7 +1129,7 @@
 	"popupShortcutNavigationMenu4": "Go to the right side of navigation. Link to page",
 	"popupShortcutNavigationMenu5": "Select option",
 	"popupShortcutNavigationPage": "Editor navigation",
-	"popupShortcutNavigationPage1": "Expand / Collapse Toggle",
+	"popupShortcutNavigationPage1": "Expand / Collapse Toggle; Check / Uncheck Checkbox",
```

Comment:
This is not correct way to add strings, it's 2 different strings one of which is already translated into 20 languages.

Reply:
Ok, there aren't any kind of guides for that, so I didn't know. Could you tell me what's the correct way of adding a new string and translations for it then?

Reply:
There is no actual guide, but if you change something it needs to be translated again and it creates less friction when it's a new line, especially with such a case when new line is added to the same place like here.

Now I think you can leave it like this, does not matter too much.

Reply:
I see. I've push new code adding a new string and leaving the old as-is. See if it helps.